dist/leaflet.css

```diff
@@ -277,8 +277,7 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	box-shadow: 0 1px 5px rgba(0,0,0,0.65);
 	border-radius: 4px;
 	}
-.leaflet-bar a,
-.leaflet-bar a:hover {

```

Comment:
Any particular reason these styles were applied both to links in the default state and `:hover`?

Reply:
If this edit is causing hesitancy by any means I'll gladly revert it to focus on the main issue.